scripts/backport.py

```diff
@@ -305,16 +316,17 @@ def __init__(self, pygithub_pr_, issue_number_):
         )
         continue
 
-    # Our general notion is that there should be one commit per PR, and if there
-    # are several, they might be fixing different issues, and we can't figure
-    # figure out which commit fixes which, and which of the fixes should be
-    # backported, so we just play safe here and refuse to backport them.
-    commit_shas = [commit.sha for commit in pr_info.pygithub_commits]
-    if len(commit_shas) > 1:
-        print(
-            f'Will not backport the PR #{original_pr.number}: "{original_pr.title}" because it contains multiple commits.'
+    # Skip the PR if it changes some particularly conflict-prone files that are
+    # better to review manually.
+    changed_files = {file.filename for file in pull.get_files()}
+    stopper_files = changed_files.intersection(
+        ["sql/updates/latest-dev.sql", "sql/updates/reverse-dev.sql"]
+    )
```

Comment:
Probably better to not automatically _merge_ it, but still create the PR and tell the OP that she/he needs to handle it. If we just ignore to backport it, it is easily missed, but if there is a PR created, it will be glaringly obvious that there is work to do.

Reply:
Done.